app/models/pdc_metadata/resource.rb

```diff
@@ -4,15 +4,16 @@ module PDCMetadata
   # https://support.datacite.org/docs/datacite-metadata-schema-v44-properties-overview
   #
   class Resource
-    attr_accessor :creators, :titles, :publisher, :publication_year, :resource_type,
+    attr_accessor :creators, :titles, :publisher, :publication_year, :resource_type, :resource_type_general,
       :description, :doi, :ark, :rights
 
-    def initialize(doi: nil, title: nil, resource_type: nil, creators: [], description: nil)
+    def initialize(doi: nil, title: nil, resource_type: nil, resource_type_general: nil, creators: [], description: nil)
       @titles = []
       @titles << PDCMetadata::Title.new(title: title) unless title.nil?
       @description = description
       @creators = creators
       @resource_type = resource_type || "Dataset"
+      @resource_type_general = resource_type_general || :dataset
```

Comment:
This one is defaulted to lowercase, and the default_resource_type_general is uppercase.  Is there a reason for the difference?